specification/security/_types/RoleMappingRule.ts

```diff
@@ -33,14 +34,11 @@ export class RoleMappingRule {
 /**
  * @variants container
```

Comment:
```suggestion
 * @variants container
 * @non_exhaustive
```
Add a non exhaustive tag insteand of an additional properties implementation.